app/services/process_output_documents.rb

```diff
@@ -10,6 +19,22 @@ def call
 
     csv = CSVRenderer.new(output_documents).render
 
-    UploadToPrintHouse.new(csv).call
+    UploadToPrintHouse.new(csv, logger).call
+  end
+
+  def call_logged(logger)
```

Comment:
![bad taste in the mouth](http://s3.amazonaws.com/rapgenius/tumblr_luc3u0nIq81qen75u.gif)